doc-examples/example-java/src/main/java/example/Application.java

```diff
@@ -1,36 +1,7 @@
 package example;
 
-import com.mysql.cj.exceptions.*;
-import io.micronaut.core.annotation.TypeHint;
 import io.micronaut.runtime.Micronaut;
 
-@TypeHint({
-        AssertionFailedException.class,
```

Comment:
Are these `@TypeHint` clases no longer needed?

Reply:
There shouldn’t have been there in a first place